Marlin/src/lcd/extui/lib/mks_ui/tft_lvgl_configuration.cpp

```diff
@@ -120,6 +118,10 @@ void tft_lvgl_init() {
 
   watchdog_refresh();     // LVGL init takes time
 
+  #if MB(MKS_ROBIN_NANO)
+    OUT_WRITE(PB0, LOW);  // HE1
```

Comment:
Sometimes this is the `FAN_PIN` and sometimes `HEATER_1_PIN`. What LVGL-related function does this line serve?

Reply:
It's not my code. It's in #20191. On my board, it's HEATER2 pin. *Maybe* it's there to disable heating element on boot? I don't think it's required, but we should ask @MKS-Sean for more details.

Reply:
In our actual test, the ROBIN_NANO_V1.2 motherboard will be opened when heater2 is started. It is dangerous to open it when updating assets

Reply:
> "the ROBIN_NANO_V1.2 motherboard will be opened…"

What do you mean by "**opened**" here?

Reply:
Sorry, I didn’t express it clearly. PB0 will be set to high level, causing the MOS FET connected to it to turn on